tests/functional/lib/helpers.js

```diff
@@ -975,30 +979,31 @@ define([
     };
   }
 
-  function openPage(context, url, readySelector) {
-    var remote = getRemote(context);
-    return remote
-      .get(require.toUrl(url))
-      .setFindTimeout(config.pageLoadTimeout)
+  function openPage(url, readySelector) {
+    return function () {
```

Comment:
Could you also have used `thenify` to do this or does that behave differently somehow?